promql/functions.go

```diff
@@ -184,6 +184,107 @@ func funcIrate(ev *evaluator, args Expressions) model.Value {
 	return resultVector
 }
 
+// doubleSVal calculates the smoothed value at index i of raw data d
+// s = computed smoothed values, b = computed trend factors, d = raw
+func doubleSVal(i int, sf, tf float64, s, b, d []float64) float64 {
+
```

Comment:
Remove empty line.  Looks weird to have blank space as the first line of a function body.